modules/apps/transfer/ibc_module.go

```diff
@@ -302,8 +302,16 @@ func (im IBCModule) OnTimeoutPacket(
 }
 
 // OnChanUpgradeInit implements the IBCModule interface
-func (im IBCModule) OnChanUpgradeInit(ctx sdk.Context, portID, channelID string, order channeltypes.Order, connectionHops []string, sequence uint64, version, previousVersion string) (string, error) {
-	return version, nil
+func (im IBCModule) OnChanUpgradeInit(ctx sdk.Context, portID, channelID string, order channeltypes.Order, connectionHops []string, sequence uint64, upgradeVersion, previousVersion string) (string, error) {
```

Comment:
dropping the previous super nit https://github.com/cosmos/ibc-go/pull/4019#discussion_r1258342517 `sequence -> upgradeSequence`.

Reply:
done, I'm unsure if we should even be passing the upgrade sequence to the app - I don't see any particular use case for it immediately